pkg/helm/actions/upgrade_release.go

```diff
@@ -0,0 +1,52 @@
+package actions
+
+import (
+	"helm.sh/helm/v3/pkg/action"
+	"helm.sh/helm/v3/pkg/chart/loader"
+	"helm.sh/helm/v3/pkg/release"
+	"helm.sh/helm/v3/pkg/storage/driver"
+)
+
+func UpgradeRelease(ns, name, url string, vals map[string]interface{}, conf *action.Configuration) (*release.Release, error) {
+	client := action.NewUpgrade(conf)
+	client.Install = true
```

Comment:
I would not try to install chart here, for that we have already have `InstallChart()` function. We should not do too many things at once.